ci/Dockerfile.nightwatch

```diff
@@ -0,0 +1,11 @@
+FROM node:alpine
+WORKDIR /app/
+
+# Install
+COPY . ./
+
+RUN rm -rf packages/client packages/graphql packages/imtr-client
```

Comment:
Can you add a comment why we do this?
There were some conflicts in the packages? Can't remember.